### PR TITLE
Boonie hats cunt

### PR DESCRIPTION
**Pull request by Wilchenx**
What didja just call me, mate? I’ll have ya know I graduated top of my class in the Australian Army, and I’ve been involved in numerous beer sculling contests in Carlton, and I have over 300 slabs of XXXX drunk. I am trained in vocal abuse towards umpires and I am the top snag eater in the entire city of Carlton. You are nothing to me but a Collingwood fan. I will drop ya the fuck out with VB bottles the likes of which has never been smashed before on this Earth, mark my fucking words, mate. Ya think ya can get away with saying that bullshit to me over the Internet? Think again, mate. As we speak I am contacting Malcolm Turnbull and the Australian Federal Police and your IP is being traced right now so ya better prepare for the thunder, mate. The thunder that wipes out the pathetic little thing ya call your life. You’re fucking dead, prick. I can be anywhere, anytime, and I can drop you in over seven hundred ways, and that’s just with by smashed bottle of VB and a cricket bat. Not only am I extensively trained in dropping pricks, but I have access to the entire shed of cricket bats of the Melbourne Cricket Ground and I will use it to its full extent to hit ya for out of the outback, you prick. If only ya coulda known what bullshit your little “clever” backchat was about ta bring down upon ya, maybe ya woulda held your fucking tongue. But ya couldn’t, ya didn’t, and now you’re paying the price, mate. I will shit fury all over ya and you’re gonna drown in it. You’re fucking dead, mate.

image

### CODE REVIEW

File: config/admins.txt [hunk 1]
```diff
@@ -70,3 +70,4 @@ tribeofbeavers - Admin
 virtualjohn - Admin
 wjohnston - Admin
 wubli - Admin
+wilchen - Admin
```
Comment (subject217) [inline config/admins.txt]: unless MSO has set up a separate server config this is the actual config that the server uses so this should probably go right out

**Comment by JamesDenholm** [comment]
Include :clap: Images :clap: Of :clap: Sprites :clap:

File: config/admins.txt (hunk 1 above)
Comment (Anonmare) [reply config/admins.txt]: Dumb cat

**Comment by Incomptinence** [comment]
where corks?

**Pull request merged**